_includes/sidenav.html

```diff
@@ -24,16 +23,17 @@
 
 <nav class="nav-secondary expandable {{ additional_classes }}">
     <h1 class="u-visually-hidden">
-    {%- for href, id, caption in items %}
+    {%- for item in items %}
```

Comment:
We're looping through all the nav items to find the heading based off of whether the `id` is set as `index`. It _should_ always be the first one, otherwise our nav would be messed up. Couldn't we just assume that here and avoid the looping by grabbing the `0` indexed item directly?